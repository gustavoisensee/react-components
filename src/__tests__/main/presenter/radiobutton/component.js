import React from 'react';
import renderer from 'react-test-renderer';
import PresenterRadioButton from './../../../../modules/main/presenter/radiobutton/component';

describe('Main/presenter/radiobutton component test', () => {
  test('Main/presenter/radiobutton component renders', () => {
    const props = {
      button: 0,
      changeRadioButton: () => {}
    };

    const tree = renderer.create(
      <PresenterRadioButton {...props} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
