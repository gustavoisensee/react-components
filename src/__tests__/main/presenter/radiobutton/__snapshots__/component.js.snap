exports[`Main/presenter/radiobutton component test Main/presenter/radiobutton component renders 1`] = `
<div
  className="content">
  <div
    className="title">
    <span>
      RadioButton
    </span>
  </div>
  <div
    className="example">
    <div
      className="filter__first">
      <input
        checked={false}
        className="radiobutton"
        id="1"
        name="radiobutton"
        onChange={[Function]}
        type="radio"
        value={1} />
      <label
        className="radiobutton__label"
        htmlFor="1">
        Test 1
      </label>
    </div>
    <div
      className="filter">
      <input
        checked={false}
        className="radiobutton"
        id="2"
        name="radiobutton"
        onChange={[Function]}
        type="radio"
        value={2} />
      <label
        className="radiobutton__label"
        htmlFor="2">
        Test 2
      </label>
    </div>
    <div
      className="filter__last">
      <input
        checked={false}
        className="radiobutton"
        id="3"
        name="radiobutton"
        onChange={[Function]}
        type="radio"
        value={3} />
      <label
        className="radiobutton__label"
        htmlFor="3">
        Test 3
      </label>
    </div>
  </div>
  <div
    className="description">
    Description
  </div>
</div>
`;
